vendor.conf

```diff
@@ -0,0 +1,73 @@
+k8s.io/kubernetes v1.6.4 https://github.com/kubernetes/kubernetes
+# https://github.com/kubernetes/client-go#compatibility-matrix
+k8s.io/client-go v3.0.0-beta.0 https://github.com/kubernetes/client-go
+k8s.io/apimachinery release-1.6 https://github.com/kubernetes/apimachinery
+k8s.io/apiserver release-1.6 https://github.com/kubernetes/apiserver
+#
+github.com/Sirupsen/logrus master
+github.com/containers/image master
```

Comment:
Yeah, I'll pin some of them, even if, if you add a new dep you don't need to run the full vndr again, just for that one you're adding.